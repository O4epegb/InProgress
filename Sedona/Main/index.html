<!DOCTYPE html>
<!--[if lt IE 7]><html lang="ru" class="lt-ie9 lt-ie8 lt-ie7"><![endif]-->
<!--[if IE 7]><html lang="ru" class="lt-ie9 lt-ie8"><![endif]-->
<!--[if IE 8]><html lang="ru" class="lt-ie9"><![endif]-->
<!--[if gt IE 8]><!-->
<html lang="ru">
<!--<![endif]-->

<head>
    <meta charset="utf-8" />
    <title>Welcome to Sedona</title>
    <meta name="description" content="" />
    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
    <link rel="shortcut icon" href="favicon.png" />
    <link rel="stylesheet" href="libs/bootstrap/bootstrap-grid-3.3.1.min.css" />
    <link rel="stylesheet" href="libs/font-awesome-4.2.0/css/font-awesome.min.css" />
    <link rel="stylesheet" href="css/fonts.css" />
    <link rel="stylesheet" href="css/main.css" />
    <link rel="stylesheet" href="css/media.css" />
</head>

<body>
    <div class="container">
        <div class="row">
            <div class="col-md-12">
                <div class="main-shadow">
                    <header class="theader">
                        <div class="container">
                            <div class="row">
                                <div class="col-md-12">
                                    <div class="theader-wrapper">
                                        <nav class="theader-menu">
                                            <ul class="theader-menu__list">
                                                <li><a class="active" href="#index.html">Информация</a>
                                                </li>
                                                <li><a href="#index.html">Фото и видео</a>
                                                </li>
                                                <li>
                                                    <a href="index.html"><img src="images/logo.png" alt="City Of Sedona">
                                                    </a>
                                                </li>
                                                <li><a href="#index.html">Карта штата</a>
                                                </li>
                                                <li><a href="hotels.html">Гостиницы</a>
                                                </li>
                                            </ul>
                                            <ul class="theader-menu__list--mobile hidden-lg">
                                                <li>
                                                    <button class="theader-menu__list--mobile-button">
                                                        <i class="fa fa-bars"></i>
                                                    </button>
                                                    <ul class="theader-menu__list--dropdown">
                                                        <li><a class="active" href="#index.html">Информация</a>
                                                        </li>
                                                        <li><a href="#index.html">Фото и видео</a>
                                                        </li>
                                                        <li><a href="#index.html">Карта штата</a>
                                                        </li>
                                                        <li><a href="hotels.html">Гостиницы</a>
                                                        </li>
                                                    </ul>

                                                </li>
                                                <li style="">
                                                    <a href="index.html"><img src="images/logo.png" alt="City Of Sedona">
                                                    </a>
                                                </li>
                                            </ul>
                                        </nav>
                                        <div class="theader-content clearfix">
                                            <div class="theader-content__text">
                                                <p>Welcome</p>
                                                <p>— to the gorgeous —</p>
                                                <h1>Sedona</h1>
                                                <p>“Because the Grand
                                                    <br>Canyon sucks!”</p>
                                            </div>
                                        </div>
                                    </div>
                                </div>
                            </div>
                        </div>
                    </header>
                    <main class="index-main">
                        <div class="features">
                            <div class="container">
                                <div class="row">
                                    <div class="col-md-12">
                                        <header class="features-header">
                                            <h2>Седона — небольшой городок в Аризоне,<br> заслуживающий большего </h2>
                                            <p>Рассмотрим 5 причин, по которым Седона круче, чем Гранд Каньон!</p>
                                        </header>
                                    </div>
                                </div>
                                <div class="row">
                                    <div class="col-md-4">
                                        <section class="features__big">
                                            <h3>Настоящий городок</h3>
                                            <p>— №1 —</p>
                                            <p>Седона не атракцион для туристов, там течет своя жизнь</p>
                                        </section>
                                    </div>
                                    <div class="col-md-8">
                                        <section class="features__big-img">
                                            <img src="images/reason1.jpg" alt="Настоящий городок">
                                        </section>
                                    </div>
                                </div>
                                <div class="row">
                                    <div class="col-md-4">
                                        <section class="features__other">
                                            <div class="features__other-img features__other-img__house"></div>
                                            <h3>Жильё</h3>
                                            <p>Рекомендуем пожить в настоящем мотеле, всё как в кино!</p>
                                        </section>
                                    </div>
                                    <div class="col-md-4">
                                        <section class="features__other">
                                            <div class="features__other-img features__other-img__burger"></div>
                                            <h3>Еда</h3>
                                            <p>Всегда заказывай фирменный бургер, вы не разочаруетесь!</p>
                                        </section>
                                    </div>
                                    <div class="col-md-4">
                                        <section class="features__other">
                                            <div class="features__other-img features__other-img__souvenir"></div>
                                            <h3>Сувениры</h3>
                                            <p>Не только китайского, но и местного производства!</p>
                                        </section>
                                    </div>
                                </div>
                                <div class="row">
                                    <div class="col-md-8">
                                        <section class="features__big-img">
                                            <img src="images/reason2.jpg" alt="Настоящий городок">
                                        </section>
                                    </div>
                                    <div class="col-md-4">
                                        <section class="features__big">
                                            <h3>Там есть мост Дьявола</h3>
                                            <p>— №2 —</p>
                                            <p>Да, по нему можно пройти!
                                                <br> Если конечно вы осмелитесь</p>
                                        </section>
                                    </div>
                                </div>
                                <div class="row">
                                    <div class="col-md-4">
                                        <section class="features__small">
                                            <h3>Небольшая площадь</h3>
                                            <p>— №3 —</p>
                                            <p>Все достопримечательности находятся очень близко</p>
                                        </section>
                                    </div>
                                    <div class="col-md-4">
                                        <section class="features__small">
                                            <h3>Красивая дорога</h3>
                                            <p>— №4 —</p>
                                            <p>Ехать в Седону из Лас-Вегаса совсем не скучно</p>
                                        </section>
                                    </div>
                                    <div class="col-md-4">
                                        <section class="features__small">
                                            <h3>Мало туристов</h3>
                                            <p>— №5 —</p>
                                            <p>Большинство едет в Гранд-Каньон и толпится там</p>
                                        </section>
                                    </div>
                                </div>
                                <div class="row">
                                    <div class="col-md-12">
                                        <section class="features-order">
                                            <h2>Заинтересовались?</h2>
                                            <p>Укажите предполагаемые даты поездки и мы покажем вам лучшие предложения гостиниц в Седоне</p>
                                            <label for="features-order__checkbox" class="features-order__label">
                                                <div class="features-order__button">
                                                    <p>Поиск <span>гостиницы в Седоне</span>
                                                    </p>
                                                </div>
                                            </label>
                                            <input type="checkbox" class="features-order__checkbox" id="features-order__checkbox">
                                            <div class="features-order__popup">
                                                <div class="features-order__popup-date">
                                                    <label class="features-order__popup-date__label" for="features-order__popup-date__in">Дата заезда:</label>
                                                    <input class="features-order__popup-date__in" type="date" name="features-order__popup-date__in" id="features-order__popup-date__in">
                                                </div>
                                                <div class="features-order__popup-date">
                                                    <label class="features-order__popup-date__label" for="features-order__popup-date__out">Дата выезда:</label>
                                                    <input class="features-order__popup-date__out" type="date" name="features-order__popup-date__out" id="features-order__popup-date__out">
                                                </div>
                                                <div class="features-order__popup-people">
                                                    <label class="order__label-people">Взрослые:
                                                        <div id="minus-adult" class="order__inner-minus"><span class="order__icon-minus"></span>
                                                        </div>
                                                        <div id="plus-adult" class="order__inner-plus"><span class="order__icon-plus"></span>
                                                        </div>
                                                        <input class="order__field-people order__field-people--adult" type="text" value="2">
                                                    </label>
                                                    <label class="order__label-people order__label-people--child">Дети:
                                                        <div id="minus-child" class="order__inner-minus__child"><span class="order__icon-minus"></span>
                                                        </div>
                                                        <div id="plus-child" class="order__inner-plus__child"><span class="order__icon-plus"></span>
                                                        </div>
                                                        <input class="order__field-people order__field-people--child" type="text" value="0">
                                                    </label>
                                                </div>
                                                <input class="features-order__popup-search" type="submit" value="Найти">

                                            </div>
                                        </section>
                                    </div>
                                </div>
                                <div class="row">
                                    <div class="col-md-12">
                                        <section class="features-map">
                                            <div id="googleMap"></div>
                                        </section>
                                    </div>
                                </div>
                            </div>
                        </div>
                    </main>
                    <footer class="main-footer">
                        <div class="container">
                            <div class="row ">
                                <div class="col-md-12">
                                    <div class="container">
                                        <div class="row">
                                            <section class="main-footer-wrapper">
                                                <div class="col-md-4">
                                                    <article class="main-footer-section">
                                                        <a class="main-footer-section__hashtag" href="#index.html">#VisitSedona</a>
                                                    </article>
                                                </div>
                                                <div class="col-md-4">
                                                    <article class="main-footer-section">
                                                        <a href="#index.html">
                                                            <div class="main-footer-social main-footer-social__twitter"></div>
                                                        </a>
                                                        <a href="#index.html">
                                                            <div class="main-footer-social main-footer-social__facebook"></div>
                                                        </a>
                                                        <a href="#index.html">
                                                            <div class="main-footer-social main-footer-social__youtube"></div>
                                                        </a>
                                                    </article>
                                                </div>
                                                <div class="col-md-4">
                                                    <article class="main-footer-section">
                                                        <p>Website by <a href="#index.html"><span class="main-footer-logos main-footer-logos__by"></span></a> for <a href="#index.html"><span class="main-footer-logos main-footer-logos__for"></span></a>
                                                        </p>
                                                    </article>
                                                </div>
                                            </section>
                                        </div>
                                    </div>
                                </div>
                            </div>
                        </div>
                    </footer>
                </div>
            </div>
        </div>
    </div>
    <!--[if lt IE 9]>
	<script src="libs/html5shiv/es5-shim.min.js"></script>
	<script src="libs/html5shiv/html5shiv.min.js"></script>
	<script src="libs/html5shiv/html5shiv-printshiv.min.js"></script>
	<script src="libs/respond/respond.min.js"></script>
	<![endif]-->
    <script src="libs/jquery/jquery-1.11.1.min.js"></script>
    <script src="https://maps.googleapis.com/maps/api/js"></script>
    <script src="js/backbtn.min.js"></script>
    <script src="js/common.js"></script>
</body>

</html>