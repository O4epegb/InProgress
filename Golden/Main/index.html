<!DOCTYPE html>
<!--[if lt IE 7]><html lang="ru" class="lt-ie9 lt-ie8 lt-ie7"><![endif]-->
<!--[if IE 7]><html lang="ru" class="lt-ie9 lt-ie8"><![endif]-->
<!--[if IE 8]><html lang="ru" class="lt-ie9"><![endif]-->
<!--[if gt IE 8]><!-->
<html lang="ru">
<!--<![endif]-->

<head>
    <meta charset="utf-8" />
    <title>Заголовок</title>
    <meta name="description" content="" />
    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
    <link rel="shortcut icon" href="favicon.png" />
    <link rel="stylesheet" href="libs/bootstrap/bootstrap-grid-3.3.1.min.css" />
    <link rel="stylesheet" href="http://maxcdn.bootstrapcdn.com/font-awesome/4.3.0/css/font-awesome.min.css">
    <link rel="stylesheet" href="css/fonts.css" />
    <link rel="stylesheet" href="css/main.css" />
    <link rel="stylesheet" href="css/media.css" />
</head>

<body>
    <div class="top-bg">
        <div class="container">
            <div class="row">
                <div class="col-md-12">
                    <header class="top-header clearfix">
                        <h1 class="logo">Golden</h1>
                        <nav class="menu">
                            <ul>
                                <li><a href="#index.html">Home</a>
                                </li>
                                <li><a href="#index.html">Services</a>
                                </li>
                                <li><a href="#index.html">Portfolio</a>
                                </li>
                                <li><a href="#index.html">About</a>
                                </li>
                                <li><a href="#index.html">Contact</a>
                                </li>
                            </ul>
                        </nav>
                    </header>
                    <h3>Welcome to our studio!</h3>
                    <h2>It's nice to meet you</h2>
                    <button class="more">Tell me more</button>
                </div>
            </div>
        </div>
    </div>
    <main>
        <section class="services">
            <div class="container">
                <div class="row">
                    <div class="col-md-12">
                        <h3>Services</h3>
                        <p>Lorem ipsum dolor sit amet, consectetur.</p>
                        <div class="row">
                            <div class="col-md-4 col-sm-6">
                                <div class="services-item">
                                    <div class="services-icon__border clearfix">
                                        <div class="services-icon services-icon__commerce"></div>
                                    </div>
                                    <a href="#index.html">E-Commerce</a>
                                    <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit. Vestibulum vehicula dictum lectus sit amet viverra. Morbi consequat nulla.</p>
                                </div>
                            </div>
                            <div class="col-md-4 col-sm-6">
                                <div class="services-item">
                                    <div class="services-icon__border">
                                        <div class="services-icon services-icon__security"></div>
                                    </div>
                                    <a href="#index.html">Web security</a>
                                    <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit. Vestibulum vehicula dictum lectus sit amet viverra. Morbi consequat nulla.</p>
                                </div>
                            </div>
                            <div class="col-md-4 col-sm-12">
                                <div class="services-item">
                                    <div class="services-icon__border">
                                        <div class="services-icon services-icon__responsive"></div>
                                    </div>
                                    <a href="#index.html">Responsive web</a>
                                    <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit. Vestibulum vehicula dictum lectus sit amet viverra. Morbi consequat nulla.</p>
                                </div>
                            </div>
                        </div>
                    </div>
                </div>
            </div>
        </section>
        <section class="portfolio">
            <div class="container">
                <div class="row">
                    <div class="col-md-12">
                        <h3>Portfolio</h3>
                        <p>Lorem ipsum dolor sit amet, consectetur.</p>
                        <div class="row">
                            <div class="col-md-4 col-sm-6">
                                <figure class="portfolio-item">
                                    <a href="#index.html"><img src="images/Portfolio-image1.jpg" alt="Ebony &amp; ivory" width="290" height="300">
                                    </a>
                                    <figcaption>
                                        <h4>Ebony &amp; ivory</h4>
                                        <p>Branding</p>
                                    </figcaption>
                                </figure>
                            </div>
                            <div class="col-md-4 col-sm-6">
                                <figure class="portfolio-item">
                                    <a href="#index.html"><img src="images/Portfolio-image2.jpg" alt="Smart stationary" width="290" height="300">
                                    </a>
                                    <figcaption>
                                        <h4>Smart stationary</h4>
                                        <p>Print design</p>
                                    </figcaption>
                                </figure>
                            </div>
                            <div class="col-md-4 col-sm-6">
                                <figure class="portfolio-item">
                                    <a href="#index.html"><img src="images/Portfolio-image3.jpg" alt="Clever poster" width="290" height="300">
                                    </a>
                                    <figcaption>
                                        <h4>Clever poster</h4>
                                        <p>Print design</p>
                                    </figcaption>
                                </figure>
                            </div>
                            <div class="col-md-4 col-sm-6">
                                <figure class="portfolio-item">
                                    <a href="#index.html"><img src="images/Portfolio-image4.jpg" alt="Vinyl record" width="290" height="300">
                                    </a>
                                    <figcaption>
                                        <h4>Vinyl record</h4>
                                        <p>Product mock-up</p>
                                    </figcaption>
                                </figure>
                            </div>
                            <div class="col-md-4 col-sm-6">
                                <figure class="portfolio-item">
                                    <a href="#index.html"><img src="images/Portfolio-image5.jpg" alt="Treehouse template" width="290" height="300">
                                    </a>
                                    <figcaption>
                                        <h4>Treehouse template</h4>
                                        <p>Web design</p>
                                    </figcaption>
                                </figure>
                            </div>
                            <div class="col-md-4 col-sm-6">
                                <figure class="portfolio-item">
                                    <a href="#index.html"><img src="images/Portfolio-image6.jpg" alt="Burned logo" width="290" height="300">
                                    </a>
                                    <figcaption>
                                        <h4>Burned logo</h4>
                                        <p>Branding</p>
                                    </figcaption>
                                </figure>
                            </div>
                        </div>
                    </div>
                </div>
            </div>
        </section>
        <section class="about">
            <div class="container">
                <div class="row">
                    <div class="col-md-12">
                        <h3>Portfolio</h3>
                        <p>Lorem ipsum dolor sit amet, consectetur.</p>
                        <div class="row">
                            <div class="col-md-4 col-sm-4 col-xs-12">
                                <article class="about-text clearfix">
                                    <h4>
                                        <span class="date">July 2010</span> <br>Our humble beginnings
                                    </h4>
                                    <p>
                                        Proin iaculis purus consequat sem cure digni ssim. Donec porttitora entum suscipit aenean rhoncus posuere odio in tincidunt. Proin iaculis purus consequat sem cure digni ssim. Donec porttitora entum suscipit aenean.
                                    </p>
                                </article>
                            </div>
                            <div class="col-md-4 col-sm-4 col-xs-12">
                                <article class="about-image">
                                    <img src="images/Image.png" alt="Beginnings">
                                </article>
                            </div>
                            <div class="col-md-4 col-sm-4 col-xs-12">
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-md-4 col-sm-4 col-xs-12">

                            </div>
                            <div class="col-md-4 col-sm-4 col-xs-12">
                                <article class="about-image">
                                    <img src="images/About-image2.png" alt="Beginnings">
                                </article>
                            </div>
                            <div class="col-md-4 col-sm-4 col-xs-12">
                                <article class="about-text about-text__left clearfix">
                                    <h4>
                                        <span class="date">January 2011</span> <br>Facing startup Battles
                                    </h4>
                                    <p>
                                        Proin iaculis purus consequat sem cure digni ssim. Donec porttitora entum suscipit aenean rhoncus posuere odio in tincidunt. Proin iaculis purus consequat sem cure digni ssim. Donec porttitora entum suscipit aenean.
                                    </p>
                                </article>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-md-4 col-sm-4 col-xs-12">
                                <article class="about-text clearfix">
                                    <h4>
                                        <span class="date">December 2012</span> <br>Enter the dark days
                                    </h4>
                                    <p>
                                        Proin iaculis purus consequat sem cure digni ssim. Donec porttitora entum suscipit aenean rhoncus posuere odio in tincidunt. Proin iaculis purus consequat sem cure digni ssim. Donec porttitora entum suscipit aenean.
                                    </p>
                                </article>
                            </div>
                            <div class="col-md-4 col-sm-4 col-xs-12">
                                <article class="about-image">
                                    <img src="images/About-image3.png" alt="Beginnings">
                                </article>
                            </div>
                            <div class="col-md-4 col-sm-4 col-xs-12">
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-md-4 col-sm-4 col-xs-12">

                            </div>
                            <div class="col-md-4 col-sm-4 col-xs-12">
                                <article class="about-image">
                                    <img src="images/About-image4.png" alt="Beginnings">
                                </article>
                            </div>
                            <div class="col-md-4 col-sm-4 col-xs-12">
                                <article class="about-text about-text__left clearfix">
                                    <h4>
                                        <span class="date">February 2014</span> <br>Our triumph
                                    </h4>
                                    <p>
                                        Proin iaculis purus consequat sem cure digni ssim. Donec porttitora entum suscipit aenean rhoncus posuere odio in tincidunt. Proin iaculis purus consequat sem cure digni ssim. Donec porttitora entum suscipit aenean.
                                    </p>
                                </article>
                            </div>
                        </div>
                        <div class="row">
                            <div class="col-md-12">
                                <article class="about-image">
                                    <img src="images/About-image5.png" alt="Beginnings">
                                    <h4 class="about-image__text">Our <br>story <br>continues <br>...</h4>
                                </article>
                            </div>
                        </div>
                    </div>
                </div>
            </div>
        </section>
        <section class="team">
            <div class="container">
                <div class="row">
                    <div class="col-md-12">
                        <h3>Our amazing team</h3>
                        <p>Lorem ipsum dolor sit amet, consectetur.</p>
                        <div class="row">
                            <div class="col-md-4 col-sm-6">
                                <div class="team-item">
                                    <img class="team-item__photo" src="images/team-member1.jpg" alt="">
                                    <h4>Kimberly Thompson</h4>
                                    <p>Marketer</p>
                                    <div class="team-social-icons">
                                        <a class="twitter-icon" href="#index.html"><i class="fa fa-twitter"></i></a>
                                        <a class="facebook-icon" href="#index.html"><i class="fa fa-facebook"></i></a>
                                        <a class="pinterest-icon" href="#index.html"><i class="fa fa-pinterest-p"></i></a>
                                    </div>
                                </div>
                            </div>
                            <div class="col-md-4 col-sm-6">
                                <div class="team-item">
                                    <img class="team-item__photo" src="images/team-member2.jpg" alt="">
                                    <h4>Rico Massimo</h4>
                                    <p>Coder</p>
                                    <div class="team-social-icons">
                                        <a class="twitter-icon" href="#index.html"><i class="fa fa-twitter"></i></a>
                                        <a class="facebook-icon" href="#index.html"><i class="fa fa-facebook"></i></a>
                                        <a class="google-icon" href="#index.html"><i class="fa fa-google-plus"></i></a>
                                    </div>
                                </div>
                            </div>
                            <div class="col-md-4 col-sm-12">
                                <div class="team-item">
                                    <img class="team-item__photo" src="images/team-member3.jpg" alt="">
                                    <h4>Uku Mason</h4>
                                    <p>Graphic Designer</p>
                                    <div class="team-social-icons">
                                        <a class="facebook-icon" href="#index.html"><i class="fa fa-facebook"></i></a>
                                        <a class="pinterest-icon" href="#index.html"><i class="fa fa-pinterest-p"></i></a>
                                    </div>
                                </div>
                            </div>
                        </div>
                        <p class="team-info">Proin iaculis purus consequat sem cure digni ssim. Donec porttitora entum suscipit aenean rhoncus posuere odio in tincidunt proin iaculis.</p>
                    </div>
                </div>
            </div>
        </section>
        <section class="partners">
            <div class="container">
                <div class="row">
                    <div class="col-md-12">
                        <a href="#index.html"><img src="images/Envato.png" alt="Envato">
                        </a>
                        <a href="#index.html"><img src="images/WordPress.png" alt="WordPress">
                        </a>
                        <a href="#index.html"><img src="images/Tuts+.png" alt="AETuts">
                        </a>
                        <a href="#index.html"><img src="images/Microlancer-copy.png" alt="Microlancer">
                        </a>
                    </div>
                </div>
            </div>
        </section>
        <section class="contact">
            <div class="container">
                <div class="row">
                    <div class="col-md-12">
                        <h3>Contact us</h3>
                        <p>Lorem ipsum dolor sit amet, consectetur.</p>
                        <form id="contactus" name="ContactUs">
                            <div class="clearfix">
                                <div class="inputs">
                                    <input type="text" required name="name" placeholder="Your Name*">
                                    <input type="email" required name="email" placeholder="Your E-mail*">
                                    <input type="text" name="subject" placeholder="Subject">
                                </div>
                                <textarea name="text" id="text" cols="0" rows="0" placeholder="Your Message"></textarea>
                            </div>
                            <input type="submit" value="Send message">
                        </form>
                    </div>
                </div>
            </div>
        </section>
    </main>
    <div class="container">
        <div class="row">
            <div class="col-md-12">
                <footer class="bottom-footer">
                    <div class="row">
                        <div class="col-md-3">
                            <p>© Copyright 2014 FreebiesXpress.com</p>
                        </div>
                    
                    <div class="col-md-6">
                        <div class="team-social-icons">
                            <a class="twitter-icon" href="#index.html"><i class="fa fa-twitter"></i></a>
                            <a class="facebook-icon" href="#index.html"><i class="fa fa-facebook"></i></a>
                            <a class="google-icon" href="#index.html"><i class="fa fa-google-plus"></i></a>
                            <a class="pinterest-icon" href="#index.html"><i class="fa fa-pinterest-p"></i></a>
                        </div>
                    </div>
                    </div>
                </footer>
            </div>
        </div>
    </div>
    <!--[if lt IE 9]>
	<script src="libs/html5shiv/es5-shim.min.js"></script>
	<script src="libs/html5shiv/html5shiv.min.js"></script>
	<script src="libs/html5shiv/html5shiv-printshiv.min.js"></script>
	<script src="libs/respond/respond.min.js"></script>
	<![endif]-->
    <script src="libs/jquery/jquery-1.11.1.min.js"></script>
    <script src="js/backbtn.min.js"></script>
    <script src="js/common.js"></script>
</body>

</html>